ansys/api/additive/v0/additive_simulation.proto

```diff
@@ -20,6 +20,8 @@ service SimulationService {
 
 	rpc DownloadFile(DownloadFileRequest) returns (stream DownloadFileResponse);
 
+	rpc DownloadLogs(DownloadLogsRequest) returns (stream DownloadLogResponse);
```

Comment:
Can we not use `google.Empy` for the request type or leave it blank? The return type should be `DownloadFileResponse`.